about/static/about.css

```diff
@@ -112,3 +114,29 @@ a.personal-link span.overlay {
 a.personal-link span.overlay:hover {
     box-shadow: 0px 0px 5px gray;
 }
+
+@media (max-width: 991px) {
+  div.member-info {
```

Comment:
I actually suggested a hardcoded CSS height to Mohamed a while back since I expected any JS solution to not work reliably. It's fine if we have to stick to this; there's a character limit on bios anyawy so cells shouldn't exceed this height.